Exercise_Lab04/Exercise_Lab04/Lab4_4/Faculty.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercise_Lab04.Lab4_4
+{
+    class Faculty : Employee
+    {
+        public int officeHours { get; set; }
+        public string rank { get; set; }
+
+        public Faculty(int officeHours, string rank, string department, float salary, int dateHired, string name, string phoneNumber, string emailAddress) : base(department, salary, dateHired, name, phoneNumber, emailAddress)
+        {
+            this.officeHours = officeHours;
```

Comment:
từ khóa `this` đại diện cho gì

Reply:
this dùng để tham chiếu đến thành phần của lớp hiện tại